src/styles/var.scss

```diff
@@ -0,0 +1,2 @@
+$duration_color: 300ms;
+$duration_transform: 300ms;
```

Comment:
Why you need two var's for one param - duration?

Reply:
ok, I fixed this